### ISSUE TITLE
关于ernie_csc损失函数的问题

### ISSUE THREAD

**Issue by xuyouqian** [shibing624/pycorrector#263]
大佬您好，在学习您的ernie_csc文本纠错模型，在这个文件
pycorrector/ernie_csc/train.py 文件中矫正的损失函数计算是这样的
corr_loss = corr_loss_act(corr_logits, corr_labels) * det_error_probs.max(axis=-1)
我理解的是检测模型检测出的信心概率越大，他的损失会越大，这是不是跟论文里的描述不一样呢。
是不是应该改成：
corr_loss = corr_loss_act(corr_logits, corr_labels) * （1-det_error_probs.max(axis=-1)）
或者：
corr_loss = corr_loss_act(corr_logits, corr_labels) * det_error_probs.min(axis=-1)

不知道我这样理解对不对？

**Comment by xuyouqian** [comment]
我好像明白了 ，没得问题了
